fastlane/lib/fastlane/actions/docs/sync_code_signing.md

```diff
@@ -91,7 +91,7 @@ username("user@fastlane.tools")
 
 _match_ also supports storing certificates of multiple teams in one repo, by using separate git branches. If you work in multiple teams, make sure to set the `git_branch` parameter to a unique value per team. From there, _match_ will automatically create and use the specified branch for you.
 
-```ruby
+```ruby-skip-tests
 match(git_branch: "team1", username: "user@team1.com")
```

Comment:
>This code sample is being evaluated as a lane as it is marked as ruby. But a Matchfile uses methods, that are not available in that content, so those have to be marked in this way.

I'm confused here 🤔 This is totally syntax that would show up in a `lane` and not in a `MatchFile` so should being marked as `ruby` the correct thing?

Reply:
You are totally right, I changed the wrong code sample. Now the `ruby-skip-tests` is for a sample that uses "options methods" outside of a lane, like it would in a `Matchfile`. 

Thanks for catching this.